src/main/resources/application.conf

```diff
@@ -37,8 +37,10 @@ encry {
     useCli = true
   }
   testing {
-    transactionGeneration = false
-    keepPoolSize = 100
+    transactionGeneration = true
```

Comment:
transactionGeneration = true switches on creating random txs on every node. Is it your goal?

Reply:
It allows you to test node properly every time you start the node (At the development stage). No doubt, we will set it to `false` in prod.